tests/Meziantou.Analyzer.Test/Rules/ClassWithEqualsTShouldImplementIEquatableTAnalyzerTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Threading.Tasks;
+using Meziantou.Analyzer.Rules;
+using Xunit;
+using TestHelper;
+
+namespace Meziantou.Analyzer.Test.Rules
+{
+    public sealed class ClassWithEqualsTShouldImplementIEquatableTAnalyzerTests
+    {
+        private static ProjectBuilder CreateProjectBuilder()
+        {
+            return new ProjectBuilder()
+                .WithAnalyzer<ClassWithEqualsTShouldImplementIEquatableTAnalyzer>();
+        }
+
+        [Theory]
+        [InlineData("public bool Equals(Test other) { return false; }", true)]
```

Comment:
I often use `throw null` to simplify tests and indicates you don't care about the method implementation. It works whatever the method type and you can use `=>`

````
public void Equals(Test other) => throw null;
public bool Equals(Test other) => throw null;
````